<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtchetSotrudniki</class>
 <widget class="QDialog" name="OtchetSotrudniki">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчёт по сотрудникам</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_criteriaOption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_columns">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Отобразить дополнительные поля</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="5">
          <widget class="QCheckBox" name="checkBox6">
           <property name="text">
            <string>Ставка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QCheckBox" name="checkBox7">
           <property name="text">
            <string>Дата рождения</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="checkBox5">
           <property name="text">
            <string>Должность</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox2">
           <property name="text">
            <string>E-mail</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox3">
           <property name="text">
            <string>Образование</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="checkBox4">
           <property name="text">
            <string>Ученое звание</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox1">
           <property name="text">
            <string>Телефоны</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox8">
           <property name="text">
            <string>Место рождения</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox9">
           <property name="text">
            <string>Национальность</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBox10">
           <property name="text">
            <string>Серия паспорта</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="checkBox11">
           <property name="text">
            <string>Адрес проживания</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="checkBox12">
           <property name="text">
            <string>Семейное положение</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QCheckBox" name="checkBox13">
           <property name="text">
            <string>№ Соц. защиты</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Должность</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_showReport">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сгенерировать отчет</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/generate.png</normaloff>:/img/img/generate.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>График</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/chart.png</normaloff>:/img/img/chart.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox_doljnost"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_sostav">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>ППС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>УВС</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Состав</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_set">
        <property name="text">
         <string>Настройки полей</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/1downarrow.png</normaloff>:/img/img/1downarrow.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_showReport</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_2</zorder>
     <zorder>comboBox_doljnost</zorder>
     <zorder>pushButton_set</zorder>
     <zorder>groupBox_columns</zorder>
     <zorder>label</zorder>
     <zorder>comboBox_sostav</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
